changelog.md

```diff
@@ -4,10 +4,10 @@ This file contains all the notable changes done to the Ballerina SQL package thr
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
 
 ## [0.6.0-beta.1] - 2021-05-11
 
 ### Added
-
 - [Introduced `ArrayValueType` type and `TypedValue` object to configure the types of an SQL array](https://github.com/ballerina-platform/ballerina-standard-library/issues/104)
 - Newly introduce TimeWithTimezoneOutParameter and TimestampWithTimezoneOutParameter for using TimeWithTimeZone Type OutParameter
```

Comment:
Lets add the changelog here also

Reply:
Added in https://github.com/ballerina-platform/module-ballerina-sql/pull/245/commits/310decd979cc682a9ffe1fc9aadf5d38d59d4510